pkg/internal/apis/certmanager/types_certificaterequest.go

```diff
@@ -33,6 +33,14 @@ const (
 	// Issued indicates that a CertificateRequest has been completed, and that
 	// the `status.certificate` field is set.
 	CertificateRequestReasonIssued = "Issued"
+
+	// Approved indicates that a CertificateRequest has been approved by the
+	// approver, and the CertificateRequest is ready for signing.
+	CertificateRequestReasonApproved = "Approved"
+
+	// Denied indicates that a CertificateRequest has been denied by the
+	// approver, and the CertificateRequest will be never be signed.
+	CertificateRequestReasonDenied = "Denied"
```

Comment:
Idea: could we add a link to the proposal so that people can know more about that aspect of the API?

```suggestion
	// Approved indicates that a CertificateRequest has been approved by the
	// approver, and the CertificateRequest is ready for signing. This condition
	// is part of the Policy API described at
	// https://github.com/jetstack/cert-manager/blob/master/design/20210302.certificate-request-policy.md
	CertificateRequestReasonApproved = "Approved"

	// Denied indicates that a CertificateRequest has been denied by the
	// approver, and the CertificateRequest will be never be signed. This
	// condition is part of the Policy API described at
	// https://github.com/jetstack/cert-manager/blob/master/design/20210302.certificate-request-policy.md
	CertificateRequestReasonDenied = "Denied"
```

Reply:
^ can we do that?

Reply:
I like this idea, and sorry I didn't comment on it in my last review. but don't want to hold things up with another round of E2E tests etc. 
@JoshVanL maybe add this  link in a followup PR.

Reply:
Sure. We need to wait for that PR to merge first though, since that link currently 404s.